app/workers/maintenance/uncache_media_worker.rb

```diff
@@ -6,7 +6,7 @@ class Maintenance::UncacheMediaWorker
   sidekiq_options queue: 'pull'
 
   def perform(media_attachment_id)
-    media = media_attachment_id.is_a?(MediaAttachment) ? media_attachment_id : MediaAttachment.find(media_attachment_id)
+    media = MediaAttachment.find(media_attachment_id)
```

Comment:
It's very slow like this. I think it's better to `select` more columns if background option is false

Reply:
Okay. Thank you for your advice.

Reply:
@Gargron  Is it okay to delete "select()" like "MediaAttachment.where.not(remote_url: '').where.not(file_file_name: nil).where('created_at < ?', time_ago).reorder(nil).find_in_batches" ?